source/LumiCalReco/src/GlobalMethodsClass.cpp

```diff
@@ -447,7 +448,8 @@ bool GlobalMethodsClass::SetGeometryDD4HEP() {
   }
 
   // layer thickness
-  GlobalParamD[ZLayerThickness] = (layers[0].inner_thickness + layers[0].outer_thickness)/dd4hep::mm;;
+  GlobalParamD[ZLayerThickness] = (layers[0].inner_thickness + layers[0].outer_thickness)/dd4hep::mm;
+  GlobalParamD[ZLayerZOffset] = (layers[0].inner_thickness + (layers[0].outer_thickness/2.))/dd4hep::mm;
```

Comment:
inner_thickness is the thickness of the layer up to the centre of the sensitive volume, outer_thickness is the thickness after the centre of the sensitive volume. I think the ZLayerZOffset should just be `-outer_thickness` so that the zCell is in the proper location

Reply:
I agree, I had confused the meaning of thicknesses. 
But since the first layer ID is 0, I think we should rather add inner_thickness than subtracting outer_thickness (would be the case if counting from 1)

Reply:
Yes, correct.